src/rohit/WhileLoop.java

```diff
@@ -0,0 +1,55 @@
+package rohit;
+
+public class WhileLoop {
+	void evennumber(int start, int end) {
+		System.out.println("Even Numbers::");
+		int index = start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?